.github/workflows/build-test-core-x86-ocaml5.jsonnet

```diff
@@ -0,0 +1,39 @@
+// coupling: This is the same than build-test-core-x86.jsonnet but for OCaml 5.
+
+local semgrep = import 'libs/semgrep.libsonnet';
+
+local core_x86 = import 'build-test-core-x86.jsonnet';
+
+// ----------------------------------------------------------------------------
+// The job
+// ----------------------------------------------------------------------------
+local job = core_x86.export.job(
+  container=semgrep.ocaml5_alpine_container,
+  artifact='ocaml-build-artifacts-ocaml5-release',
+  // TODO: some tests are currently failing with OCaml5! but at least
```

Comment:
@IagoAbal I've disabled the tests for ocaml5 for now.
but at least we're checking whether semgrep builds with ocaml5.